packages/frontend/components/Navigation.tsx

```diff
@@ -0,0 +1,98 @@
+import styles from '../styles/Navigation.module.scss';
+import NavigationItem from './Navigation/NavigationItem';
+
+enum CircleX {
+    Home = 208,
+    Clips = 291,
+    Messages = 374,
+    Soundboard = 457,
+    Credits = 540
+}
+
+interface NavProps {
+    title: string,
+    page: string
+}
+
+function Navigation({ title, page }:NavProps) {
+    return (
+    <div className={styles.navigation}>
+        <div className={styles.name}>
+            <svg viewBox="0 0 431 221" xmlns="http://www.w3.org/2000/svg">
+                <g>
+                    <path d="M0 221V0H395L340.571 131.04L0 221Z" fill="#FF7B39" fillOpacity="0.61"/>
+                </g>
+                <g>
+                    <path d="M-26 198V-29H431L378.5 105.598L-26 198Z" fill="#FF7B39"/>
+                </g>
+                <text x="40%" y="50%" dominantBaseline="middle" textAnchor="middle">
+                    桐生ココ
+                </text>
+            </svg>
+        </div>
+        <div className={styles.navigators}>
+            <svg viewBox="0 0 631 126" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <g>
+                    <path d="M42 0H642V108.898L196.889 126L42 0Z" fill="#FF7B39" fillOpacity="0.57"/>
+                </g>
+                <g>
+                    <path d="M0 -20H650V88.8983L145.013 106L0 -20Z" fill="#FF7B39" />
+                </g>   
+                <g>
+                    <NavigationItem
+                        href="/"
+                        x="189"
+                        y="26"
+                        width="38"
+                        height="40"
+                        path="M204.984 26.5735C205.829 25.8637 206.897 25.4746 208 25.4746C209.103 25.4746 210.171 25.8637 211.016 26.5735L225.078 38.3985C225.602 38.8383 226.022 39.3874 226.311 40.0071C226.6 40.6268 226.75 41.3022 226.75 41.986V61.6266C226.75 62.4554 226.421 63.2502 225.835 63.8363C225.249 64.4223 224.454 64.7516 223.625 64.7516H215.031C214.202 64.7516 213.408 64.4223 212.822 63.8363C212.235 63.2502 211.906 62.4554 211.906 61.6266V50.6891H204.094V61.6266C204.094 62.4554 203.765 63.2502 203.178 63.8363C202.592 64.4223 201.798 64.7516 200.969 64.7516H192.375C191.546 64.7516 190.751 64.4223 190.165 63.8363C189.579 63.2502 189.25 62.4554 189.25 61.6266V41.9844C189.25 41.3007 189.4 40.6253 189.689 40.0055C189.978 39.3858 190.398 38.8368 190.922 38.3969L204.984 26.5719V26.5735ZM209.006 28.9641C208.725 28.727 208.368 28.5969 208 28.5969C207.632 28.5969 207.275 28.727 206.994 28.9641L192.931 40.7922C192.757 40.9385 192.618 41.121 192.522 41.3269C192.425 41.5328 192.375 41.7572 192.375 41.9844V61.625H200.969V50.6875C200.969 49.8587 201.298 49.0639 201.884 48.4778C202.47 47.8918 203.265 47.5625 204.094 47.5625H211.906C212.735 47.5625 213.53 47.8918 214.116 48.4778C214.702 49.0639 215.031 49.8587 215.031 50.6875V61.625H223.625V41.9844C223.625 41.7567 223.575 41.5317 223.479 41.3252C223.383 41.1187 223.243 40.9357 223.069 40.7891L209.006 28.9641Z" />
```

Comment:
Do you think you could make these into actual react components with .svg file imports in front of the svg? It would probably make it easier to move stuff around and reduce the clutter of having the entire svg path in there. Also helps with accessibility. If you don't want to or can't ill do that eventually so don't worry about it ^^

Reply:
Ok, I'll try that